extensions/positron-r/resources/js/htmlwidget.js

```diff
@@ -52,54 +52,52 @@ const arrayify = (obj) => {
  *  loaded.
  */
 const renderDependencies = (dependencies) => {
-	return new Promise((resolve, _reject) => {
```

Comment:
This change is to ensure that scripts are appended _and_ loaded in sequence. I'm not sure why the browser wasn't already doing that, but this change does seem to fix highcharter and leaflet widgets.